test/unittests/parser_test.cpp

```diff
@@ -390,7 +398,7 @@ TEST(parser, export_multiple)
 
 TEST(parser, start)
 {
-    const auto section_contents = bytes{} + uint8_t{0x07};
+    const auto section_contents = bytes{0x07};
```

Comment:
You can shorten many cases like this:

```suggestion
    const auto bin = wasm_prefix + make_section(8, {0x07});
```